documentation/asciidoc/accessories/camera/filters.adoc

```diff
@@ -30,47 +30,41 @@ The HQ and GS Cameras use a Hoya CM500 infrared filter. Its transmission charact
 
 image::images/hoyacm500.png[CM500 Transmission Graph,width="65%"]
 
-== Filter Removal
+== IR Filter
 
-NOTE: This procedure applies to both the HQ and GS cameras. 
-
-WARNING: *This procedure cannot be reversed:* the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. *Removing it will void the warranty on the product*. Nevertheless, removing the filter will be desirable to some users.
+Both the High Quality Camera and Global Shutter Camera contain an IR filter, which is used to reduce the camera's sensitivity to infrared light. This ensures that outdoor photos look more natural. However, some nature photography can be enhanced with the removal of this filter; the colours of sky, plants, and water can be affected by its removal. The camera can also be used without the filter for night vision in a location that is illuminated with infrared light.
 
-image:images/FILTER_ON_small.jpg[width="65%"]
+=== Filter Removal
 
-Both the High Quality Camera and Global Shutter Camera contain an IR filter, which is used to reduce the camera's sensitivity to infrared light. This ensures that outdoor photos look more natural. However, some nature photography can be enhanced with the removal of this filter; the colours of sky, plants, and water can be affected by its removal. The camera can also be used without the filter for night vision in a location that is illuminated with infrared light.
+The filter removal procedure applies to both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
 
 WARNING: Before proceeding read through all of the steps and decide whether you are willing to void your warranty. *Do not proceed* unless you are sure that you are willing to void your warranty.
 
-To remove the filter:
+WARNING: *This procedure cannot be reversed:* the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. *Removing it will void the warranty on the product*. Nevertheless, removing the filter will be desirable to some users.
```

Comment:
I strongly suggest we don't put two admonition blocks up against each other, especially the same type. We should probably use less in general.

Personally, I would delete the first warning (on line 41). The second warning covers that info in a shorter, clearer way anyway.